src/main.rs

```diff
@@ -926,8 +964,17 @@ async fn run() -> Result<(), Error> {
                 Err(err) => event!(Level::ERROR, err = %err, "Pod reconciliation error."),
             }
         });
+
     let eip_controller = Controller::new(eip_api, ListParams::default())
         .run(reconcile_eip, on_error, context)
+        .then(|rr| async {
+            if rr.is_ok() {
+                if let Err(e) = report_eip_quota_status(&ec3_client, &quota_client).await {
+                    event!(Level::WARN, err = %e, "Could not report on EIP / quota status");
```

Comment:
I feel it would be better to `#[instrument]` the `report_eip_quota_status` function and let the automatic error annotation handle the error-reporting instead of handling it as a warning here (which turns invisible in the trace view)

Reply:
Addressed in https://github.com/MaterializeInc/k8s-eip-operator/pull/145/commits/4ac62caaa94cd5132633de107db310886665f788